### PR TITLE
Make AzureIndexer compatible with new Stratis.Bitcoin packages

### PR DESCRIPTION
**Pull request by quantumagi**
This PR updates the AzureIndexer to make it compatible with the latest Stratis.Bitcoin packages.

### CODE REVIEW

File: AzureIndexer/Stratis.IndexerD/Program.cs [hunk 1]
```diff
@@ -15,26 +15,20 @@ public class Program
...
                 .UseNodeSettings(nodeSettings)
-                .UseStratisConsensus()
+                .UsePosConsensus()
                 .UseBlockStore()
                 .UseAzureIndexer()
                 .Build();
 
-            node.RunAsync().Wait();
+            if (node != null)
+                node.RunAsync().Wait();
```
Comment (bokobza) [inline AzureIndexer/Stratis.IndexerD/Program.cs]: You can even remove the wait and make the Main method async like we did in the other projects.

**Comment by bokobza** [comment]
Is that the only changes that are required?

**Comment by quantumagi** [comment]
Those are the only changes required to achieve compatibility with the new version of the packages. The new code has been running for a few hours now on the server.

**Comment by bokobza** [comment]
ok I've pushed it in, can you please have a look at my comment in a new PR?
Thanks Gustave!

File: AzureIndexer/Stratis.IndexerD/Program.cs (hunk 1 above)
Comment (quantumagi) [reply AzureIndexer/Stratis.IndexerD/Program.cs]: See PR https://github.com/stratisproject/BlockExplorer/pull/44.

**Comment by quantumagi** [comment]
@bokobza, please see PR https://github.com/stratisproject/BlockExplorer/pull/44.

**Pull request merged**